config/initializers/admin_constants.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+SUPER_ADMIN = 'super_admin'
```

Comment:
SUPER_ADMIN = 'super_admin'.freeze
ADMIN       = 'admin'.freeze
ROW_USER    = 'user'.freeze

should be in model